Makefile

```diff
@@ -28,6 +28,9 @@ else
 	export INDEX_URL_BUILD_ARG ?= PIP_INDEX_URL
 endif
 
+# Use this variable for the docker run command
+DOCKER_REGISTRY_ARG=$(if [ "$(DOCKER_REGISTRY)" != "" ]; then echo "$(DOCKER_REGISTRY)"; fi)
```

Comment:
Why not just reuse `$DOCKER_REGISTRY` in this case?

Reply:
I couldn't use it since the original `export DOCKER_REGISTRY ?= ""` would lead to `""yelp/openapi-generator-cli:20201026` and cause the github actions to fail

Reply:
! I think that indicates we've been setting `DOCKER_REGISTRY` incorrectly if so; i think the intent is not to have the explicit string `""`, but an empty string.

Reply:
If we have to do this two-docker-registry arg thing, technically I would swap these variable names -- the one we're calling `DOCKER_REGISTRY` is actually being passed around as our itests/yelp_package dockerfiles' `DOCKER_REGISTRY` `ARG`s :D